File: Javascript/7. 10부제 (하)/AA.js
function solution(day, arr){
    let answer=0
    let cnt=0
    for(let i=0;i<arr.length;i++){
        if (arr[i]%10==day){
            cnt++
        }
    }
    answer=cnt
    return answer
}

arr=[12, 20, 54, 30, 87, 91, 30]
console.log(solution(0, arr))

File: Python/섹션 2 (완) - 코드 구현력 기르기/10. 점수 계산 (하)/AA.py
import sys
#sys.stdin=open('input.txt', 'rt')

n=int(input())
arr=list(map(int,input().split()))

cnt=0
result=0
for i in range(len(arr)):
    if arr[i]==1:
        cnt+=1
        result+=cnt
    else:
        cnt=0

print(result)